From e2ce47e2cb85306bcf57160ae3c4a9026cb28018 Mon Sep 17 00:00:00 2001
From: sha-rath <64174696+sha-rath@users.noreply.github.com>
Date: Tue, 22 Mar 2022 20:05:24 +0530
Subject: Changes log level to debug while fetching AWS info and typo fix
 (#38040)

* changed log level to warn while fetching aws info and typo fix

* changed to debugf
---
 pkg/bootstrap/platform/aws.go       | 2 +-
 pkg/bootstrap/platform/discovery.go | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/pkg/bootstrap/platform/aws.go b/pkg/bootstrap/platform/aws.go
index 54e882d7be..af48e826a9 100644
--- a/pkg/bootstrap/platform/aws.go
+++ b/pkg/bootstrap/platform/aws.go
@@ -98,7 +98,7 @@ func getAWSInfo(path string, ipv6 bool) (string, error) {
 
 	resp, err := http.DoHTTPGetWithTimeout(url, time.Millisecond*100)
 	if err != nil {
-		log.Errorf("error in getting aws info for %s : %v", path, err)
+		log.Debugf("error in getting aws info for %s : %v", path, err)
 		return "", err
 	}
 	return resp.String(), nil
diff --git a/pkg/bootstrap/platform/discovery.go b/pkg/bootstrap/platform/discovery.go
index 1f061587da..2ca902b4f6 100644
--- a/pkg/bootstrap/platform/discovery.go
+++ b/pkg/bootstrap/platform/discovery.go
@@ -28,8 +28,8 @@
 	numPlatforms   = 3
 )
 
-var CloudPlatform = env.RegisterStringVar("CLOUD_PLATFORM", "", "Clound Platform on which proxy is running, if not specified,"+
-	"Istio will try to discover the platform. Valid platform values aws,azure,gcp,none.").Get()
+var CloudPlatform = env.RegisterStringVar("CLOUD_PLATFORM", "", "Cloud Platform on which proxy is running, if not specified, "+
+	"Istio will try to discover the platform. Valid platform values are aws, azure, gcp, none").Get()
 
 // Discover attempts to discover the host platform, defaulting to
 // `Unknown` if a platform cannot be discovered.
-- 
2.35.3

